modules/gitops-argo-cd-properties.adoc

```diff
@@ -29,51 +32,78 @@ a|* _<Processors.Operation>_ - The number of operation processors.
   * _<AppSync>_ - AppSync is used to control the sync frequency of Argo CD applications
   * _<Sharding.enabled>_ - Enable sharding on the Argo CD Application Controller component. This property is used to manage a large number of clusters to relieve memory pressure on the controller component.
   * _<Sharding.replicas>_ - The number of replicas that will be used to support sharding of the Argo CD Application Controller.
+  * _<Sharding.dynamicScalingEnabled>_ - Enable dynamic scaling of the Argo CD Application Controller component. This property allows the Operator to scale the number of replicas based on the number of clusters being managed by the controller component. Setting this property to `true` will override the configuration of `Sharding.enabled` and `Sharding.replicas`.
+  * _<Sharding.minShards>_ - The minimum number of replicas of the Argo CD Application Controller.
+  * _<Sharding.maxShards>_ - The maximum number of replicas of the Argo CD Application Controller.
+  * _<Sharding.clusterPerShard>_ - The number of clusters that need to be managed by each shard. When the replica count has reached the `maxShards`, the shards will manage more than one cluster.
   * _<Env>_ - Environment to set for the application controller workloads.
+
 |`DisableAdmin`    |Disables the built-in admin user.|`false` |
+
 |`GATrackingID`    |Use a Google Analytics tracking ID.|`__<empty>__` |
+
 |`GAAnonymizeusers`    |Enable hashed usernames sent to google analytics.|`false` |
```

Comment:
Google and analytics should have initial cap, like in the previous line.

Reply:
Noted.
Addressed. thank you.